qiskit/transpiler/passes/scheduling/padding/base_padding.py

```diff
@@ -107,16 +123,19 @@ def run(self, dag: DAGCircuit):
 
                     # Fill idle time with some sequence
                     if t0 - idle_after[bit] > 0:
-                        # Find previous node on the wire, i.e. always the latest node on the wire
-                        prev_node = next(new_dag.predecessors(new_dag.output_map[bit]))
-                        self._pad(
-                            dag=new_dag,
-                            qubit=bit,
-                            t_start=idle_after[bit],
-                            t_end=t0,
-                            next_node=node,
-                            prev_node=prev_node,
-                        )
+                        if self.target is None or self.target.instruction_supported(
+                            "delay", qargs=(bit_indices[bit],)
+                        ):
```

Comment:
When I started to look it this after opening #9993 I was thinking here we'd need to also handle the gates in DD sequences here besides just delays. But I think as a first step and fixing what we support in `transpile()`  this enough. Maybe for the DD padding pass we do the additional validation of the extra sequence instructions inside the pass (as a second follow-ups PR)?